src/valid/mod.rs

```diff
@@ -310,6 +313,8 @@ impl Validator {
         &mut self,
         module: &crate::Module,
     ) -> Result<ModuleInfo, WithSpan<ValidationError>> {
+        Self::validate_module_handles(module).map_err(|e| ValidationError::from(e).with_span())?;
```

Comment:
I think this should go after the `reset` and `reset_types` calls, and be conditional on the `"validate"` feature. When that's turned off, panicking is just fine. :D

Reply:
Changed as suggested in 4b1165fb. I feel silly for not considering the question of `cfg`'ing things, given the surrounding code. 😓